sailfish-rbc/src/abraham/worker.rs

```diff
@@ -372,7 +383,7 @@ impl<T: Clone + Committable + Serialize + DeserializeOwned> Worker<T> {
             .copied();
         self.comm.remove(old.collect()).await?;
         self.comm.assign(Role::Active, committee.parties().copied().collect()).await?;
-        self.config.committee_id = round.committee();
+        self.round = (round, e);
```

Comment:
The `Worker::config` field has been removed and the config is no longer modified by the worker. Instead the relevant fields are pulled on on construction and here we use the existing round information to also track the committee ID.